lib/fetching/fetching_hash.rb

```diff
@@ -21,7 +21,7 @@ def make_methods
     end
 
     def method_missing(key, *_args, &_block)
-      raise NoMethodError, "#{key} not found\nyou have:\n#{@table.inspect}"
+      raise NoMethodError, "#{key} not found\nyou have:\n#{@table.keys.join(', ')}"
```

Comment:
I don't like "you have:" anymore. Are any of these better?
- "these are available:"
- "the following methods would work, though:"
- "perhaps you meant:"
- "consider:"
- "but the data structure does have:"